.github/workflows/debezium-workflow.yml

```diff
@@ -299,6 +299,12 @@ jobs:
           restore-keys: |
             maven-debezium-test-build-${{ hashFiles('**/pom.xml') }}
 
+      - name: Add MongoDB test container hosts to /etc/hosts
+        run: >
+          for host in test-mongo{1..3} test-mongos{1..3} test-mongo-configdb{1..3} test-shard{1..3}-replica{1..3}; do 
+            echo -e "127.0.0.1\t$host" | sudo tee -a /etc/hosts; 
+          done
```

Comment:
Unfortunately, this is required so that multi-node replica sets can properly resolve. While it is possible to use IPs in the `rs.initialize` call, this is not possible on Docker Desktop for Mac as the `docker0` bridge network is not exposed. Note that this is not currently a problem for single node replica sets in CI / local using the `docker-maven-plugin` since `localhost` is valid in this setup across Mac and Linux. While there are other techniques that can be used to make this more transparent (e.g. [JVM DNS hijacking](https://github.com/CorfuDB/corfudb-cloud/blob/9e14d15a7119ef22767d0946d12faa9bfa541e30/universe/universe-core/src/main/java/org/corfudb/universe/infrastructure/docker/universe/FakeDns.java), this seemed the easiest and most consistent / reliable. However, for local testing it does require adding these host names to the local `/etc/hosts`.

Reply:
It actually si a problem for the current container run by maven as it is initialised with IP :). But it should be possible to change.